src/components/SingleTeamComponent.tsx

```diff
@@ -3,120 +3,118 @@ import { HoverWrapperComponent } from "src/components/HoverWrapperComponent";
 import * as Data from "src/data/data";
 import { Group, Person } from "src/data/types";
 
-// One box in the grid of teams, containing team names, directors, team picture and performance link
-export class SingleTeamComponent extends React.Component<any> {
-  public render() {
-    let team: Group = this.props.team;
-    let directorsList: React.ReactNode[] = [];
 
-    // Generate a string of all of the directors
-    if (team.positionIds) {
-      for (let directorIdx in team.positionIds) {
-        let directorKey = team.positionIds[directorIdx];
-        let pp = Data.PersonPositions[directorKey];
-        let dirPersonKey: string =
-          Data.PersonPositions[directorKey].personIds[0];
-        let directorPerson: Person = Data.People[dirPersonKey];
-        directorsList.push(
-          pp.picture ?
-            <div className="director">
-              <h6>{directorPerson.name}</h6>
-              <HoverWrapperComponent pp={pp} hoverDisplay={true}>
-                {pp.picture ? (
-                  <img
-                    className="board--img"
-                    src={pp.picture[0].thumbnails.large.url}
-                    width="200px"
-                    height="200px"
-                  />
-                ) : (
-                    <img
-                      className="board--img"
-                      src="afx2.png"
-                      width="200px"
-                      height="200px"
-                      style={{ opacity: 0.3 }}
-                    />
-                  )}
-              &nbsp;&nbsp;&nbsp;
-            </HoverWrapperComponent>
-            </div> :
-            <div className="director">
-              <h6>{directorPerson.name}</h6>
-              <HoverWrapperComponent pp={pp} hoverDisplay={true}>
+export const SingleTeamComponent: React.FC<any> = (props) => {
```

Comment:
Please specify the type of the props passed in. Like
`export const SingleTeamComponent: React.FC<{team: Group}> = ({team}) => {`